x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/ColumnExtractOperatorTests.java

```diff
@@ -33,7 +33,8 @@ protected SourceOperator simpleInput(int end) {
 
     record FirstWord(int channelA) implements ColumnExtractOperator.Evaluator {
         @Override
-        public void computeRow(BytesRef input, Block.Builder[] target) {
+        public void computeRow(BytesRefBlock inputBlock, int row, Block.Builder[] target, BytesRef spare) {
+            BytesRef input = inputBlock.getBytesRef(row, spare);
```

Comment:
I've been using the word "index" or just `i` for these indices created by getting the position.